Data/future/comex/margins/AUP.csv

```diff
@@ -1,73 +1,1014 @@
+20160722,128,116
+20160725,90,81
+20160726,77,70
+20160727,64,58
+20160728,51,46
+20160729,39,35
+20160801,396,360
+20160802,383,348
+20160803,371,337
+20160804,358,325
+20160805,346,314
+20160808,307,279
+20160809,294,267
+20160810,281,255
+20160811,269,244
+20160812,256,232
+20160815,217,197
+20160816,205,186
+20160817,192,174
+20160818,180,163
+20160819,167,151
+20160822,128,116
+20160823,116,105
+20160824,103,93
+20160825,90,81
+20160826,77,70
+20160829,39,35
+20160830,26,23
+20160831,14,12
+20160901,396,360
+20160905,370,336
+20160906,357,324
+20160907,344,312
+20160908,330,300
+20160909,317,288
+20160912,278,252
+20160913,264,240
+20160914,251,228
+20160915,238,216
+20160916,225,204
+20160919,185,168
+20160920,172,156
+20160921,159,144
+20160922,146,132
+20160923,132,120
+20160926,93,84
+20160927,80,72
+20160928,66,60
+20160929,53,48
+20160930,40,36
+20161003,371,337
+20161004,358,325
+20161005,346,314
+20161006,333,302
+20161007,319,290
+20161010,281,255
+20161011,269,244
+20161012,256,232
+20161013,244,221
+20161014,230,209
+20161017,192,174
+20161018,180,163
+20161019,167,151
+20161020,153,139
+20161021,141,128
+20161024,103,93
+20161025,90,81
+20161026,77,70
+20161027,64,58
+20161028,51,46
+20161031,14,12
+20161101,396,360
+20161102,383,348
+20161103,370,336
+20161104,357,324
+20161107,317,288
+20161108,304,276
+20161109,291,264
+20161110,278,252
+20161111,264,240
+20161114,225,204
+20161115,212,192
+20161116,198,180
+20161117,185,168
+20161118,172,156
+20161121,132,120
+20161122,119,108
+20161123,106,96
+20161124,93,84
+20161125,80,72
+20161128,40,36
+20161129,27,24
+20161130,14,12
+20161201,396,360
+20161205,371,337
+20161206,358,325
+20161207,346,314
+20161208,333,302
+20161209,319,290
+20161212,281,255
+20161213,269,244
+20161214,256,232
+20161215,244,221
+20161216,230,209
+20161219,192,174
+20161220,180,163
+20161221,167,151
+20161222,153,139
+20161223,141,128
+20161226,103,93
+20161227,90,81
+20161228,77,70
+20161229,64,58
+20161230,51,46
+20170102,14,12
+20170103,371,337
+20170104,358,325
+20170105,346,314
+20170106,333,302
+20170109,294,267
+20170110,281,255
+20170111,269,244
+20170112,256,232
+20170113,216,196
+20170116,182,165
+20170117,171,155
+20170118,160,145
+20170119,148,134
+20170120,137,124
+20170123,103,93
+20170124,92,83
+20170125,80,72
+20170126,69,62
+20170127,58,52
+20170130,24,21
+20170131,11,10
+20170201,352,320
+20170202,340,309
+20170203,327,297
+20170206,290,263
+20170207,277,251
+20170208,264,240
+20170209,252,229
+20170210,239,217
+20170213,202,183
+20170214,189,171
+20170215,176,160
+20170216,164,149
+20170217,151,137
+20170220,114,103
+20170221,101,91
+20170222,88,80
+20170223,76,69
+20170224,63,57
+20170227,26,23
+20170228,13,11
+20170301,352,320
 20170303,385,350
-# https://www.cmegroup.com/notices/clearing/2017/05/Chadv17-183.html
-20170519,363,330
-# https://www.cmegroup.com/notices/clearing/2017/11/Chadv17-467.html
+20170306,348,316
+20170307,336,305
+20170308,324,294
+20170309,311,282
```

Comment:
Wonder if we should increase this to every month minimum. Every 3 days is a little much for this data